chia/cmds/dao.py

```diff
@@ -0,0 +1,958 @@
+from __future__ import annotations
+
+import asyncio
+from typing import Optional
+
+import click
+
+from chia.cmds.plotnft import validate_fee
+
+
+@click.group("dao", short_help="Create, manage or show state of DAOs", no_args_is_help=True)
+@click.pass_context
+def dao_cmd(ctx: click.Context) -> None:
+    pass
+
+
+# ----------------------------------------------------------------------------------------
+# ADD
+
+
+@dao_cmd.command("add", short_help="Create a wallet for an existing DAO", no_args_is_help=True)
+@click.option(
+    "-wp",
+    "--wallet-rpc-port",
+    help="Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml",
+    type=int,
+    default=None,
+)
+@click.option("-f", "--fingerprint", help="Set the fingerprint to specify which key to use", type=int)
+@click.option("-n", "--name", help="Set the DAO wallet name", type=str)
+@click.option(
+    "-t",
+    "--treasury-id",
+    help="The Treasury ID of the DAO you want to track",
+    type=str,
+    required=True,
+)
+@click.option(
+    "-fa",
```

Comment:
I've removed the multi-letter options. For DAO creation and rule updates I've left only the long-form options available since having a bunch of random letters is more confusing. And since these commands aren't going to be used terribly frequently I feel like this isn't a bad trade off